pypoptools/pypopminer/entities.cpp

```diff
@@ -152,4 +152,10 @@ void init_entities() {
       .add_property(
           "difficulty", &VbkBlock::getDifficulty, &VbkBlock::setDifficulty)
       .add_property("nonce", &VbkBlock::getNonce, &VbkBlock::setNonce);
+
+  class_<PopData, boost::shared_ptr<PopData>>("PopData")
+      .def("__repr__", &PopData::toPrettyString)
+      .def_readonly("atvs", &PopData::atvs)
+      .def_readonly("vtbs", &PopData::vtbs)
+      .def_readonly("context", &PopData::context);
```

Comment:
add a method `serialize()` which will return a list of 3 lists:
```
[["0ab31..12"], ["0ab31..12"], ["0ab31..12"]]
```

Reply:
you can do that if you define a function
```
list Serialie(const PopData& self) {
 ...
}
```

then add Serialize here

```
.def("serialize", &Serialize)
```

Reply:
Added